kyt.config.js

```diff
@@ -10,8 +10,14 @@ module.exports = {
     // modify baseConfig as needed
     const jestConfig = Object.assign({}, baseConfig);
 
+    // When running the end-to-end tests change the testRegex value to match
+    // related tests
+    if (process.env.TEST_ENV && process.env.TEST_ENV === 'e2e') {
```

Comment:
The test command from kyt do not accept different config files other than the default.